library/templates/dob_now_permits.yml

```diff
@@ -0,0 +1,35 @@
+dataset:
+  name: &name dob_now_permits
+  version: "{{ version }}"
+  acl: public-read
+  source:
+    script: *name
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+    geometry:
+      SRS: null
+      type: NONE
+
+  destination:
+    name: *name
+    geometry:
+      SRS: null
+      type: NONE
+    options:
+      - "OVERWRITE=YES"
+      - "PRECISION=NO"
+    fields: []
+    sql: null
+
+  info:
+    description: |
+      DOB NOW job applications received from DOB via FTP
```

Comment:
This description could be improved I think as it says we get the files via FTP but then they are stored on digital ocean. I think detail on how the files go up to edm-private would be helpful for me at this moment and thus helpful for future readers